scania-digital-design-system/sdds
# feat(storybook): deprecate CSS/Native components

**Describe pull-request**  
In move towards the final removal of Native(css) components for the Tegel package we start with separating them in Storybook. 

**Solving issue**  
Fixes:[DTS-1469](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?assignee=63ff6d6c314f5088138187f0&selectedIssue=DTS-1469)

**How to test**  
Pull down the branch and build it and run storybook and click around and make sure that everything works. 

[DTS-1469]: https://tegel.atlassian.net/browse/DTS-1469?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
